lib/message.rb

```diff
@@ -0,0 +1,69 @@
+require 'pg'
+
+class Message
+  attr_reader :id, :name, :message
+
+  def initialize(id:, name:, message:)
+    @id = id
+    @name = name
+    @message = message
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
```

Comment:
This if statement is repeated in every function is there a way you could refactor your code to make it DRY?